random_data.go

```diff
@@ -324,11 +324,12 @@ func Month() string {
 // FullDate returns full date
 func FullDate() string {
 	timestamp := time.Now()
-	day := Day()
-	month := Month()
 	year := timestamp.Year()
-	fullDate := day + " " + strconv.Itoa(Number(1, 30)) + " " + month[0:3] + " " + strconv.Itoa(year)
-	return fullDate
+	month := Number(1, 13)
+	maxDay := time.Date(year, time.Month(month), 1, 0, 0, 0, 0, time.UTC).AddDate(0, 1, -1).Day()
```

Comment:
Would this be clearer as `maxDay := time.Date(year, time.Month(month + 1), 0, 0, 0, 0, 0, time.UTC).Day()`?
https://play.golang.org/p/Xe6Us-aWh3F